### PR TITLE
Fixed Typographical Error in guide/english/voice/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/voice/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Voice
 # Voice
 Speech recognition allows users affected by accessibility difficulties (such as permanent visual impairment or temporary impairment while driving) the ability to navigate content on a website or input text data (such as a form). 
 
-Speech synthesis provides websites the ability to provide information to users by reading text.
```
Comment (ahtee) [inline guide/english/voice/index.md]: I prefer this sentence as it is less wordy and still gets the same message across

File: guide/english/voice/index.md [hunk 2]
```diff
@@ -81,10 +81,10 @@ The audio captured by the Echo will be influenced by:
 4) the output audio against the wall of the room.
 
 ## Software
-The software components within the platform include both Natural Language Understanding (NLU) as well as Automated Speech Recognition (ASR).  These software components can be leveraged by custom written "skills" by independent software developers who are then certified to a set of standards by Amazon. There are already more than 20k of these custom skills available through their app store.
```
Comment (ahtee) [inline guide/english/voice/index.md]: same with this

File: guide/english/voice/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/english/voice/index.md]: I agree with @ahtee

Comment (iamrajiv) [reply guide/english/voice/index.md]: thanks, @ahtee for suggestions

**Review by ahtee — APPROVED** [review]

**Comment by iamrajiv** [comment]
@ahtee @thecodingaviator is this PR good to be merged

**Comment by thecodingaviator** [comment]
@iamrajiv Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**